### DISCUSSION TITLE
Question: How does sorting work with getJobs function

### DISCUSSION THREAD

**Discussion by epretha** [OptimalBits/bull#2111] [Q&A]
Hi,
     When we call the `getJobs` to get a list of completed jobs, on a queue, will the jobs be first retrieved from the completed list and then sorted or vice versa ?  

Say we have a queue with 5000 completed jobs. I want to get the latest 2 completed jobs i.e. [job#5000, job#4999] . So which one of the following calls will give the correct list of jobs in the above given order ? 

`getJobs(['completed'], 0, 1, false)`

`getJobs(['completed'], 4998, 4999, false)`

Thanks.

**Comment by manast** [comment]
Use ```queue.getCompleted(0, 1);```

**Reply by epretha** [reply]
So, if I wanted to use `getJobs` method instead of `getCompleted`, how should the call look like ?

`getJobs(['completed'], 0, 1, false)` ?

Thanks.

**Reply by manast** [reply]
https://github.com/OptimalBits/bull/blob/develop/lib/getters.js#L120

**Reply by epretha** [reply]
Thanks!
